index.html

```diff
@@ -88,7 +95,8 @@ <h4 class="social-media-title">Let's Connect</h4>
           <div class="project-img">
             <a href="#">
               <img
-                src="images/about-image/tonic-project.png"
+                src="/images/about-image/Snapshoot_Portfolio.png"
```

Comment:
- [ ] Kindly Fix the background color for the cards to be the same as the Figma design🙏🏻 instead of white backgound